manage.py

```diff
@@ -1,24 +1,23 @@
 #!/usr/bin/env python
 import os
 
-# Edit this if necessary or override the variable in your environment.
-os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'fjord.settings')
-
 from fjord import manage_utils
 
-manage_utils.setup_environ(__file__)
-
-from fjord.base import monkeypatches
-monkeypatches.patch()
 
+if __name__ == "__main__":
+    os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'fjord.settings')
 
-SKIP_CHECK = os.environ.get('SKIP_CHECK', '0')
-USING_VENDOR = os.environ.get('USING_VENDOR', '0')
+    print 'Using %s' % os.environ['DJANGO_SETTINGS_MODULE']
```

Comment:
I figured out that `./manage.py runserver` spawns another process, one for web serving another file file watching. It is goofy, but not a lot that we can do.